draft-ietf-ppm-dap.md

```diff
@@ -3784,6 +3784,23 @@ involving crafted DAP task parameters can be mitigated by having the Aggregators
 refuse shared parameters that are trivially insecure (e.g., a minimum batch size
 of 1 report).
 
+### Predictable or Enumerable Task IDs
+
+This specification imposes no requirements on task IDs except that they be
+globally unique. One way to achieve this is to use random task IDs, but
+deployments can also use schemes like {{?I-D.draft-ietf-ppm-dap-taskprov-01}}
```

Comment:
I'm a little worried about this dependency, but I suppose it's likely we'll do Last Call for both DAP and taskprov around the same time.

Reply:
We could omit the reference, but then I think it becomes unhelpfully vague. Let's see what happens as we make our way through LC and the rest of the review process.